dev/backport/.cherry_picker.toml

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+team = "apache"
+repo = "airflow"
+"check_sha"= "a85d94e6cdcd09efe93c3acee0b4ce5c9508bc23"
```

Comment:
This is one of the squashed commits merged into the main branch. It ensures the script is authenticating against the correct repository during execution. ref: https://github.com/apache/airflow/commit/a85d94e6cdcd09efe93c3acee0b4ce5c9508bc23